### PR TITLE
Change project types to staticLib

### PR DESCRIPTION
**Pull request by swift-kim**
- Change project types and remove linker-specific properties according to the tool change in https://github.com/flutter-tizen/flutter-tizen/pull/195. (A deprecation warning will be generated if the project type is not updated.)
- Change python scripts file modes to 755.

### CODE REVIEW

**Comment by swift-kim** [comment]
The CI is broken :(

File: packages/messageport/tizen/project_def.prop [hunk 1]
```diff
@@ -2,31 +2,21 @@
 # for details.
 
 APPNAME = messageport_tizen_plugin
-type = sharedLib
-profile = common-5.5
+type = staticLib
+profile = common-4.0
```
Comment (swift-kim) [inline packages/messageport/tizen/project_def.prop:6]: This change is intentional.

**Comment by HakkyuKim** [comment]
The problem is caused by: https://github.com/flutter-tizen/plugins/blob/92364110fd5d0bb16ab3d1fb803b9c24f403d865/.github/workflows/build.yml#L16-L18

which should have been removed. I'll fix it.

**Comment by WonyoungChoi** [comment]
it should be rebased for checks green.

**Comment by HakkyuKim** [comment]
@swift-kim 
The #176 just merged to master, it will work after rebase :)

**Review by HakkyuKim — APPROVED** [review]

**Pull request merged**